schema.graphql

```diff
@@ -93,4 +97,10 @@ type User @entity {
   totalDeposits: BigInt!
   "User's total withdrawals"
   totalWithdrawals: BigInt!
+}
+
+type StrategyReportInVaultAPRReport @entity {
```

Comment:
Intermediary entity that links strategy reports and vault apr reports in a many-to-many relationship